src/Fixer/Semicolon/SpaceAfterSemicolonFixer.php

```diff
@@ -44,6 +44,14 @@ public function getDefinition()
     /**
      * {@inheritdoc}
      */
+    public function getPriority()
+    {
+        return -1;
```

Comment:
please put comment with reasoning ("should run before/after XXX")

Reply:
:+1: totally forgot that :)